src/pages/ExamsPage/components/ExternalReviewDashboard.jsx

```diff
@@ -1,13 +1,35 @@
 import React from 'react';
 import PropTypes from 'prop-types';
 
+import { Button } from '@edx/paragon';
+import { useIntl } from '@edx/frontend-platform/i18n';
 import { getLaunchUrlByExamId } from '../utils';
+import messages from '../messages';
 
-const ExternalReviewDashboard = ({ exam }) => (
-  <div data-testid="review_dash">
-    {exam && <iframe title="lti_tool" src={getLaunchUrlByExamId(exam.id)} width="100%" height="1100" style={{ border: 'none' }} />}
-  </div>
-);
+const ltiToolEmbed = false; // This is set to false for now but will eventually need to be a configurable variable.
+
+const ExternalReviewDashboard = ({ exam }) => {
+  const { formatMessage } = useIntl();
+
+  // By default, launch instructor tool in a new tab. Otherwise, embed in dashboard as iframe.
+  return (
+    <div data-testid="review_dash">
+      <div>
+        {!ltiToolEmbed && exam && (
+          // todo: add spacing
+          // todo: ask about noopener and aria from example
```

Comment:
Zach had suggested using this as an example: https://github.com/openedx/edx-platform/blob/8e93280ec48bfd7c71cef060111070d2c4854b5b/lms/templates/lti.html#L37

I'm specifically wondering whether I need to include `rel="noopener"` or `aria-hidden="true"`. Reading about noopener, it seems like we don't want this for this case because we want to allow auth for this new page. I wasn't sure about aria, as I see it's some type of accessibility feature. But it seems like in this case, we do want this to be readable and not hidden.

Reply:
I think rel=noopener is unnecessary with a modern web browser and the aria-hidden is just hiding the font awesome icon from the screen reader. That said, maybe we want an icon here as well?

Reply:
Added a launch icon from the paragon icons!